sentry_sdk/api.py

```diff
@@ -53,6 +53,9 @@ def overload(x):
     "set_user",
     "set_level",
     "set_measurement",
+    "get_current_span",
+    "get_current_transaction",
+    "get_current_span_or_transaction",
```

Comment:
you also have to add these to `__init__.py`

Reply:
done.